src/meta-client/src/lib.rs

```diff
@@ -74,15 +76,23 @@ fn default_metadata_cache_tti() -> Duration {
     Duration::from_secs(300u64)
 }
 
+fn default_tcp_nodelay() -> bool {
```

Comment:
nitpick: it seems unnecessary to provide `default_xxx()` for setting a default value for each field in `MetaClientOptions`.